sound/soc/sof/core.c

```diff
@@ -315,6 +334,11 @@ static int sof_probe(struct platform_device *pdev)
 		goto comp_err;
 	}
 
+	if (plat_data->fedev == 1) {
```

Comment:
why are you checking this, please add a comment.

Reply:
OK, i will add comments in the code.